public/locales/es/translation.json

```diff
@@ -101,6 +101,19 @@
     "howDoesItWork": "¿Como funciona?",
     "tos": "Términos de servicio"
   },
+  "home": {
+    "resourcesDescription": "Encuentra recursos verificados por la comunidad para tus necesidades actuales",
+    "resourcesTitle": "Buscar recursos de reingreso",
+    "rateMyPoDescription": "Comparte tu experiencia de forma anónima",
+    "rateMyPoTitle": "Califica a tu PO",
+    "recentResources": "Recursos recientes",
+    "title": "Inicio",
+    "voteDescription": "Tienes el derecho de votar por tus derechos.",
+    "voteTitle": "Regístrate para votar",
+    "welcomeMessage": "Recursos, apoyo y experiencias compartidas para reingreso.",
```

Comment:
"...durante el reingreso." (to reflect the English; also, definite article required here)